common/config/rush/version-policies.json

```diff
@@ -1,27 +1,32 @@
-[

```

Comment:
it looks like probably the line endings changed on this file, I can rebase and undo the change if needed